spaceship.zsh

```diff
@@ -24,6 +24,7 @@ SPACESHIP_PREFIX_NVM="${SPACESHIP_PREFIX_NVM:-$SPACESHIP_PREFIX_ENV_DEFAULT}"
 SPACESHIP_PREFIX_RUBY="${SPACESHIP_PREFIX_RUBY:-$SPACESHIP_PREFIX_ENV_DEFAULT}"
 SPACESHIP_PREFIX_SWIFT="${SPACESHIP_PREFIX_SWIFT:-$SPACESHIP_PREFIX_ENV_DEFAULT}"
 SPACESHIP_PREFIX_GOLANG="${SPACESHIP_PREFIX_GOLANG:-$SPACESHIP_PREFIX_ENV_DEFAULT}"
+SPACESHIP_PREFIX_DOCKER="${SPACESHIP_PREFIX_DOCKER:-$SPACESHIP_PREFIX_GIT}"
```

Comment:
Consider not having `$SPACESHIP_PREFIX_GIT` as reference. Let it be just `" on "`, because that's not ok to bind it to Git character.